R/package.R

```diff
@@ -37,15 +39,27 @@
 #' })
 #'
 #' prefix
+#'
+#' # modify tolower to fail if given a non-character
+#' tolower <- argufy(base::tolower, x = ~ is.character)
```

Comment:
True, I guess was trying to show it Error to piece the modification
worked.  Could have surrounded with testthat::expect_error() I suppose...

In R/package.R
<https://github.com/gaborcsardi/argufy/pull/4#discussion_r36694138>:

> @@ -37,15 +39,27 @@
>  #' })
>  #'
>  #' prefix
> +#'
> +#' # modify tolower to fail if given a non-character
> +#' tolower <- argufy(base::tolower, x = ~ is.character)

Maybe if you do as.character instead, then the example runs. Not a big
deal, though.

—
Reply to this email directly or view it on GitHub
<https://github.com/gaborcsardi/argufy/pull/4/files#r36694138>.